deps.edn

```diff
@@ -551,16 +551,16 @@
   ;; https://github.com/djblue/portal
 
   :inspect/portal-cli
-  {:extra-deps {djblue/portal {:mvn/version "0.23.0"}}}
+  {:extra-deps {djblue/portal {:mvn/version "0.26.0"}}}
 
   :inspect/portal-web
-  {:extra-deps {djblue/portal             {:mvn/version "0.23.0"}
-                org.clojure/clojurescript {:mvn/version "1.10.844"}}
+  {:extra-deps {djblue/portal             {:mvn/version "0.26.0"}
+                org.clojure/clojurescript {:mvn/version "1.11.54"}}
```

Comment:
TODO: ClojureScript version will need to be tested with Portal to ensure there are no issues.  Portal is still showing ClojureScript 1.10.844, although the README may simple have not been updated.